Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
@en-dash cool, sounds like [`en-dash/PyBioC`](https://github.com/en-dash/PyBioC) is the superior choice.

What's the syntax for the iterative writing of documents?